src/redismodule.h

```diff
@@ -620,6 +646,9 @@ static const RedisModuleEvent
 #define REDISMODULE_SUBEVENT_EVENTLOOP_AFTER_SLEEP 1
 #define _REDISMODULE_SUBEVENT_EVENTLOOP_NEXT 2
 
+#define REDISMODULE_SUBEVENT_KEY_DELETED 0
```

Comment:
the module is not allowed to change the dataset (cause propagation) within this event right? should we document it somewhere?